riot/riot.py

```diff
@@ -213,13 +214,48 @@ def run_suites(
             finally:
                 results.append(result)
 
-        print("\n-------------------summary-------------------", file=out)
+        def is_warning(output):
```

Comment:
```suggestion
       @staticmethod
        def is_warning(output):
```

Reply:
I realize my original comment was weird because the method is actually a closure.
There's no need to define a closure here (i.e. to re-define the function on every execution of this method), so you should move it outside the method as a method :)

Reply:
@Isabella-Pham this still needs to be addressed 🙂